ros1_bridge/__init__.py

```diff
@@ -818,9 +818,9 @@ def load_ros2_message(ros2_msg):
         message_relative_path += '.msg'
     else:
         raise RuntimeError(
-            f"message '{ros2_msg.package_name}/msg/{ros2_msg.message_name}' "
-            f"was not found in prefix '{ros2_msg.prefix_path}' with either "
-            f"file extension '.msg' or '.idl'")
+            "message '%s/msg/%s' " % (ros2_msg.package_name, ros2_msg.message_name) +
+            "was not found in prefix '%s' with either " % (ros2_msg.prefix_path) +
+            "file extension '.msg' or '.idl'")
```

Comment:
```suggestion
            "message '%s/msg/%s' was not found in prefix '%s' with either"
            "file extension '.msg' or '.idl' %
            (ros2_msg.package_name, ros2_msg.message_name, ros2_msg.prefix_path))
```

(I think that is a lot easier to read)